.eslintrc.js

```diff
@@ -18,4 +18,12 @@ module.exports = {
   },
   root: true,
   ignorePatterns: ["**/*.js", "node_modules"],
+  overrides: [
+    {
+      files: ["src/bin/**"],
+      rules: {
+        "custom-rules/valid-constructors": 0,
```

Comment:
This custom linting rule is designed to enforce [conventions](https://github.com/guardian/cdk/blob/main/docs/architecture-decision-records/002-component-constuctors.md) for the constructs/patterns which are published as part of this library (e.g. `GuPlayApp`), in order to improve consistency for users. 

It is not relevant to the `cli` part of this repo (since users never work with those classes directly), so this change turns it off for irrelevant files/directories.